### PR TITLE
support skip_test option in yaml file

### PR DESCRIPTION
**Pull request by iyabchen**
https://github.com/hashicorp/terraform-provider-google/issues/13623

Support a skip_test option in the yaml file, similar to skip_vcr. 
If skip_test var is not an empty string, then the test is generated with a t.Skip("xxx"). 

**Release Note Template for Downstream PRs (will be copied)**

See [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) for guidance.

```release-note:none

```

### CODE REVIEW

**Comment by ScottSuarez** [comment]
Reassigning to @melinath to ensure this is the way to go. Not sure if there are additional documentation needed around this.

File: mmv1/api/resource/examples.go [hunk 1]
```diff
@@ -149,6 +149,9 @@ type Examples struct {
 	// Or a config with two fine grained resources that have a race condition during create
 	SkipVcr bool `yaml:"skip_vcr,omitempty"`
 
+	// The reason to skip a test. Only when this is not empty then the test is skipped.
```
Comment (melinath) [inline mmv1/api/resource/examples.go:152]:
```suggestion
	// The reason to skip a test. For example, a link to a ticket explaining the issue that needs to be resolved before 
	// unskipping the test. If this is not empty, the test will be skipped.
```

**Review by melinath — APPROVED** [review]
This seems reasonable to me. I don't think we have an obvious place for docs about this to live at the moment but we can add that separately / later.

cc @shuyama1

**Review by melinath — CHANGES_REQUESTED** [review]
looks like gofmt is failing

**Comment by iyabchen** [comment]
> looks like gofmt is failing

Done, seems some trailing whitespace on the comment line.

**Review by melinath — APPROVED** [review]

**Pull request merged**